webapp/src/pages/FoodList.js

```diff
@@ -87,14 +87,7 @@ function Product({ product, offeringId }) {
   return (
     <Col xs={6} className="mb-4">
       <div className="position-relative">
-        <SumaImage
-          image={images[0]}
-          alt={name}
-          params={{ crop: "attention" }}
```

Comment:
Why did you change the crop behavior?

Reply:
I had added that crop param but it didn't make sense so I left it at the default "center".